packages/ui/src/theme/createTheme.ts

```diff
@@ -63,6 +63,29 @@ function setupTokens(obj: any, path = []) {
   return tokens;
 }
 
+/**
+ * This function takes in an array of WebDesignTokens and filters out the duplicate WebDesignTokens
+ * that are defined in the DEPRECATED_TOKENS list.
+ * @param tokens
+ * @returns WebDesignTokens[]
+ */
+function removeDeprecated(tokens: WebDesignToken[]): WebDesignToken[] {
+  let tokenList = tokens;
+  DEPRECATED_TOKENS.forEach((duplicateToken) => {
+    const filteredTokens = tokenList.filter(
+      (token) => duplicateToken.tokenName === token.name
+    );
+    if (filteredTokens.length >= 2) {
```

Comment:
Why should this block not run if `filteredTokens` has a length of `1`?

Reply:
Here we are running on 2 because we are expecting 1 to always exist and the point of this code block is to choose the desired token based on the static path value, and remove all the other duplicate tokens.